### DISCUSSION TITLE
Where did it all go wrong when import Notifications from sp_api.api？

### DISCUSSION THREAD

**Discussion by xiaomile** [saleweaver/python-amazon-sp-api#709] [Q&A] [answered]
Python 3.6.8 (default, Nov 16 2020, 16:55:22)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sp_api.api import Notifications
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/sp_api/api/__init__.py", line 4, in <module>
    from .product_fees.product_fees import ProductFees
  File "/usr/local/lib/python3.6/site-packages/sp_api/api/product_fees/product_fees.py", line 7, in <module>
    class ProductFees(Client):
  File "/usr/local/lib/python3.6/site-packages/sp_api/api/product_fees/product_fees.py", line 99, in ProductFees
    def get_product_fees_estimate(self, estimate_requests: list[dict]) -> ApiResponse:
TypeError: 'type' object is not subscriptable
![image](https://user-images.githubusercontent.com/14927720/186888961-5e215887-42ed-4180-92fd-c1312c891bb1.png)

**Comment by sushmitapandit** [comment]
Please fix this issue
Imports are failing for TypeError: 'type' object is not subscriptable 

![image](https://user-images.githubusercontent.com/34543004/187061388-059d7f90-cf8a-435d-891f-9a558bdc949a.png)

**Comment by sushmitapandit** [comment, marked as answer]
Fix is:
This needs Python version greater than equal to 3.9

**Comment by bryce-pilcher** [comment]
The root issue is that in 3.9 the built-in `list` became usable in generic contexts whereas before `List` had to be imported from `typing`.  This was utilized in this library which caused the sudden issue with older versions of Python.  Although, this has been reverted to `List` in this [commit](https://github.com/saleweaver/python-amazon-sp-api/commit/e06e56ead7b5f8dcdf2b479cbdd6d2afd568593d) and is in available in the latest version of this library.  So you don't have to update to Python 3.9 to use this, you can just update this library.
